arch/RISCV/RISCVInstPrinter.c

```diff
@@ -0,0 +1,183 @@
+//===-- RISCVInstPrinter.cpp - Convert RISCV MCInst to asm syntax ---------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// This class prints an RISCV MCInst to a .s file.
+//
+//===----------------------------------------------------------------------===//
+
+#ifdef CAPSTONE_HAS_RISCV
+
+#include <stdio.h>		// DEBUG
+#include <stdlib.h>
+#include <string.h>
+#include <capstone/platform.h>
+
+#include "RISCVInstPrinter.h"
+#include "RISCVBaseInfo.h"
+#include "../../MCInst.h"
+#include "../../SStream.h"
+#include "../../MCRegisterInfo.h"
+#include "../../utils.h"
+#include "RISCVMapping.h"
+
+//#include "RISCVDisassembler.h"
+
+#define GET_REGINFO_ENUM
+#define GET_REGINFO_MC_DESC
+#include "RISCVGenRegisterInfo.inc"
+#define GET_INSTRINFO_ENUM
+#include "RISCVGenInstrInfo.inc"
+
+// Autogenerated by tblgen.
+static void printInstruction(MCInst *MI, SStream *O, MCRegisterInfo *MRI);
+static bool printAliasInstr(MCInst *MI, SStream *OS, void *info);
+static void printOperand(MCInst *MI, unsigned OpNo, SStream *O);
+static void printFenceArg(MCInst *MI, unsigned OpNo, SStream *O);
+static void printCSRSystemRegister(const MCInst*, unsigned, SStream *);
+static void printFRMArg(MCInst *MI, unsigned OpNo, SStream *O);
+static void printCustomAliasOperand( MCInst *, unsigned, unsigned, SStream *);
+/// getRegisterName - This method is automatically generated by tblgen
+/// from the register set description.  This returns the assembler name
+/// for the specified register.
+static const char *getRegisterName(unsigned RegNo, unsigned AltIdx);
+
+// Include the auto-generated portion of the assembly writer.
+#define PRINT_ALIAS_INSTR
+#include "RISCVGenAsmWriter.inc"
+
+/// NULL
+void RISCV_post_printer(csh ud, cs_insn *insn, char *insn_asm, MCInst *mci) 
+{
+  	assert(0 && "RISCV post-printer doesn't implementated.");
```

Comment:
if this is unused, we dont need to have this function. just dont setup it in RISCVModule.c, all will be fine.

Reply:
RISCVModule.c had been set NULL.
This is unused code, can be deleted.